syftlib/src/main/java/org/openmined/syft/framework/PlanExecutor.kt

```diff
@@ -0,0 +1,24 @@
+package org.openmined.syft.framework
+
+import android.content.Context
+import com.google.protobuf.ByteString
+import org.pytorch.Module
+import java.io.File
+import java.io.FileOutputStream
+
+class PlanExecutor {
+
```

Comment:
I made a plan class for our purposes in the previous PR but am not sure whether we would need to override the syft-proto plan class with our custom execute implementations

Reply:
I need to think about this. It was still part of the draft to put things together and see that the deserialization was happening. I'll be back to it and consider your alternative.